### ISSUE TITLE
Javascript handler

### ISSUE THREAD

**Issue by pawamoy** [pawamoy/mkdocstrings#92]
**Is your feature request related to a problem? Please describe.**
Opening for visibility.

**Describe the solution you'd like**
A Javascript handler.

**Additional context**
Could use [jsdoc](https://github.com/jsdoc/jsdoc)

**Comment by pawamoy** [comment]
Would you try and send a PR for it :smile: ?
A handler can be quite easy to write, especially if there's a tool like jsdoc that you can use to retrieve documentation from the source code :slightly_smiling_face:

**Comment by tbsexton** [comment]
Sure, will have to dig a bit to learn how it works. Any starting pts or pitfalls to watch for?

**Comment by tbsexton** [comment]
also, noticing that e.g. the crystal handler is split out into it's own repo. Any reason this should be a pr? When would a handler be a pr v.s. a separate repo?

**Comment by pawamoy** [comment]
Yes sorry, you're right, it should preferably live in its own repository, just like the Crystal handler!

**Comment by pawamoy** [comment]
As for starting points, take a look at the docs here: https://mkdocstrings.github.io/handlers/overview/#custom-handlers

**Comment by oprypin** [comment]
You can also come over to the chat. https://gitter.im/mkdocstrings/community
Good for some quick questions so you don't have to dig through the code yourself.

**Comment by tbsexton** [comment]
@oprypin will do. 

@pawamoy ok, so i can either make the package for you to fork into the mkdocstrings organization, or you can start the barebones/skeleton of the repo and I'm happy to start a draft PR into it. Personally would prefer the latter to reduce possible confusion to other collaborators.

**Comment by pawamoy** [comment]
Sure, we'll set that up for you then :) It should be done this evening (in 10 hours)
